### PR TITLE
ASoC: Intel: boards: Add support for ALC1019 speaker amp

### PR DESCRIPTION
**Pull request by yongzhi1**
This patch adds basic support for ALC1019 to common module.

Signed-off-by: Yong Zhi <yong.zhi@intel.com>

### CODE REVIEW

File: sound/soc/intel/boards/sof_realtek_common.c [hunk 1]
```diff
@@ -498,5 +499,86 @@ void sof_rt1019p_dai_link(struct snd_soc_dai_link *link)
...
+	link->ops = &rt1019_ops;
+}
+EXPORT_SYMBOL_NS(sof_rt1019_dai_link, SND_SOC_INTEL_SOF_REALTEK_COMMON);
+
+void sof_rt1019_codec_conf(struct snd_soc_card *card)
+{
+	card->codec_conf = rt1019_amp_conf;
+	card->num_configs = ARRAY_SIZE(rt1019_amp_conf);
+}
+EXPORT_SYMBOL_NS(sof_rt1019_codec_conf, SND_SOC_INTEL_SOF_REALTEK_COMMON);
```
Comment (plbossart) [inline sound/soc/intel/boards/sof_realtek_common.c:581]: this patch looks fine in isolation, but who uses the two symbols defined? Looks like we're missing a patch in this PR?

Comment (yongzhi1) [reply sound/soc/intel/boards/sof_realtek_common.c:581]: We tested the spk amp alone with vendor EVL kit at BCLK=64fs, so I am thinking to add the machine driver changes when the full AIC (hs+spk) becomes available.

Comment (bardliao) [reply sound/soc/intel/boards/sof_realtek_common.c:581]: > this patch looks fine in isolation, but who uses the two symbols defined? Looks like we're missing a patch in this PR?

They will be used by sof_rt5682.c

Comment (plbossart) [reply sound/soc/intel/boards/sof_realtek_common.c:581]: @yongzhi1 it's easier to review when the functionality is all present, that way we can submit the patches upstream in one shot. Let's keep this on the back-burner for now and revisit what everything is ready.

Comment (yongzhi1) [reply sound/soc/intel/boards/sof_realtek_common.c:581]: Ack, sure, let me change to draft.

**Comment by plbossart** [comment]
@yongzhi1 can you remind me if we need this PR or if it can be closed?

**Comment by plbossart** [comment]
@yongzhi1 ping. I'll close this by the end of the week unless you want to keep this open.

**Comment by yongzhi1** [comment]
> @yongzhi1 ping. I'll close this by the end of the week unless you want to keep this open.

Ack, sorry for the delay, the ALC1019 TDM mode verification was deprioritized, so let me close this one now.

**Pull request closed without merging**